pages/advanced-algorithms/built-in-graph-algorithms.mdx

```diff
@@ -372,15 +412,22 @@ To obtain all relationship on all shortest paths, use the `relationships` functi
 ```cypher
 MATCH path=(n {id: 0})-[relationships:CloseTo *ALLSHORTEST (r, n | n.total_USD)]-(m {id: 51})
 UNWIND (relationships(path)) AS edge
-RETURN DISTINCT edge; 
+RETURN DISTINCT edge;
 ```
 
-To get the total weight, add a variable at the end of the expansion expression: 
+To get the total weight, add a variable at the end of the expansion expression:
 ```cypher
 MATCH path=(n {id: 0})-[relationships:CloseTo *ALLSHORTEST (r, n | 1) total_weight]-(m {id: 9})
 RETURN nodes(path), total_weight;
 ```
 
+Remember that in the case when weight is taken from the node property and edge property, `coalesce` function must be used with edge property. It's required because when Memgraph calculates weight for the first node, there is no edge in calculation.
+For example,
```

Comment:
```suggestion
When the weight is taken from the node property and relationship property, the `coalesce()` function must be used with the relationship property. It's required because when Memgraph calculates the weight for the first node, there is no relationship in the calculation.
For example:
```